src/main/java/mediator/Mediator.java

```diff
@@ -66,7 +66,6 @@ public static Mediator getInstance() {
 	 */
 	private void printRegisteredRadioUnits() {
 		List<ManagedRadioUnit> radioUnits = radioUnitRegistry.getAllRadios();
-		radioUnits.forEach(System.out::println);
 		if (radioUnits.size() == 0) {
 			System.out.println("[ERROR] No RUs have been registered with the system.");
 		} else {

```

Comment:
It doesn't show up initially in the "files changed" view, but if you expand it downwards, the second print call is in the else statement.